include/fluent-bit/flb_output.h

```diff
@@ -678,6 +850,55 @@ static inline int flb_output_coros_size(struct flb_output_instance *ins)
     return size;
 }
 
+/* Used in engine flb_running_count */
+static inline int flb_output_timer_coros_size(struct flb_output_instance *ins)
+{
+    int size = 0;
+
+    if (flb_output_is_threaded(ins) == FLB_TRUE) {
+        /*
+         * On threaded mode, we need to count the active co-routines of
+         * every running thread of the thread pool.
+         */
+        size = flb_output_thread_pool_timer_coros_size(ins);
+    }
+    else {
+        size = mk_list_size(&ins->timer_coro_list);
+    }
+
+    return size;
+}
+
+static inline void flb_timer_coros_print(struct mk_list *timer_coro_list)
+{ 
+    struct flb_output_timer_coro *timer_coro;
+    struct mk_list *tmp;
+    struct mk_list *head;
+    int n = mk_list_size(timer_coro_list);
+    if (n != 0) {
+        /* get one coro for the job_name */
+        mk_list_foreach_safe(head, tmp, timer_coro_list) {
+            timer_coro = mk_list_entry(head, struct flb_output_timer_coro, _head);
+            if (timer_coro != NULL) {
+                flb_info("[task]   output=%s still running %d %s(s)",
```

Comment:
The plan is this will say "output=s3.1 still running 1 Upload(s)"

It won't actually be accurate tho it looks nice. A timer coroutine is not the same as an upload, each one could send many uploads. 

Each timer coro is just a thread/helper to peform uploads. 

I can't think of a better word to use here tho.